emva1288/process/results.py

```diff
@@ -812,8 +964,8 @@ def DSNU1288_DN(self):
 
         Defined as the DSNU in e^- multiplied by the overall system gain.
         Returns NaN if the dark signal variance is negative.
-
         Returns
+
```

Comment:
Blank line above `Returns` instead of below.

Reply:
opps, sorry